src/box/lua/key_def.c

```diff
@@ -273,14 +273,14 @@ lbox_key_def_gc(struct lua_State *L)
  * Raise error otherwise.
  */
 static int
-lbox_key_def_extract_key(struct lua_State *L)
+lbox_key_def_extract_key(struct lua_State *L, int idx, const char *obj_name)
```

Comment:
It is not strictly followed across the code, but AFAIR there is a convention that `lbox_` prefixed functions adhere the [lua_CFunction](https://www.lua.org/manual/5.1/manual.html#lua_CFunction) prototype (so they can be passed to [lua_pushcfunction](https://www.lua.org/manual/5.1/manual.html#lua_pushcfunction) and exposed to the Lua land). Other functions that need the Lua state are usually prefixed with `luaT_` and they can accept more than just `struct lua_State *`.

I don't like `box` in this `lbox_`, to be honest, but such formalism is better than lack of any style rules or mixing different ones.

@igormunkin, can you, please, approve or deny this statement?

Reply:
I don't know when `lbox_` prefix should be used. What about `luaT_` prefix: IIRC `T` is for Tarantool specifics, but these functions can have lua_CFunction signature or any other.

Reply:
Renamed